### ISSUE TITLE
Install Tensorflow for Gitlab-CI

### ISSUE THREAD

**Issue by BElluu** [tensorflow/tensorflow#35219]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Gitlab-CI, I think that's Linux
- TensorFlow version: 2.0.0
- Python version: 3.8
- Installed using virtualenv? pip? conda?: pip

**Describe the problem**

I need to add Tensorflow to my requirement.txt. First I added "tensorflow" without a specific version, but in Gitlab-CI linters give me information that I need to add a specific version, so I added it and it looks like "tensorflow == 2.0.0-rc0" but in Gitlab-CI I got that:

```
$ pip install -r requirements.txt
Collecting numpy==1.16.4
  Downloading https://files.pythonhosted.org/packages/d3/4b/f9f4b96c0b1ba43d28a5bdc4b64f0b9d3fbcf31313a51bc766942866a7c7/numpy-1.16.4.zip (5.1MB)
ERROR: Could not find a version that satisfies the requirement tensorflow==2.0.0-rc0 (from -r requirements.txt (line 2)) (from versions: none)
ERROR: No matching distribution found for tensorflow==2.0.0-rc0 (from -r requirements.txt (line 2))
```

**Provide the exact sequence of commands / steps that you executed before running into the problem**

1. Create .Gitlab-CI.yml
2. Code for that file:
```
image: "python:3.8"

before_script:
  - python --version
  - python -c 'import struct;print( 8 * struct.calcsize("P"))'
  - pip install --upgrade pip
  - pip install --upgrade setuptools
  - pip install -r requirements.txt
```

3. Add file requirements.txt with that:
```
numpy==1.16.4
tensorflow==2.0.0-rc0
```
4. Push branch to repo
5. CI give me error like above.

**Any other info / logs**
tensorflow==2.0.0rc0 and tensorflow==2.0.0 or tensorflow==2.0.0-rc0 give me same error.

**Comment by BElluu** [comment]
Change image from Python 3.8 for Python 3.7 work for me. So problem is with latest Python 3.8

**Comment by ravikyram** [comment]
@BElluu 
We can expect python 3.8 support with TF 2.2.See [#33374 (comment)](https://github.com/tensorflow/tensorflow/issues/33374#issuecomment-562663993).Can i close this issue as the issue resolved by downgrading to Python 3.7. Thanks!
